MaterialComponent.js

```diff
@@ -60,7 +60,12 @@ export default class MaterialComponent extends Component {
   render() {
     this.buildClassName();
     // Fetch a VNode
-    const element = this.materialDom(this.props);
+    const componentProps = this.props;
+    if (componentProps.class){
```

Comment:
Is this condition really necessary? Might as well just call delete every time I think.

Reply:
may be.. just a cautious check TBH